packages-nextgen/kestrel_core/src/kestrel/frontend/mapping/entity/stix.yaml

```diff
@@ -0,0 +1,24 @@
+# TODO mapping for artifact:
+autonomous-system: organization
+directory: file
+domain-name: endpoint
+email-addr: user
+email-message: email
+file: file
+ipv4-addr: endpoint
+ipv6-addr: endpoint
+mac-addr: endpoint
+network-traffic: network-activity
```

Comment:
I am rethinking about whether we should use dash or underscore. Looks like the URLs of all activities uses underscore, and we may better do the underscore.

We can also auto-normalize dash to underscore in Kestrel parser.

Reply:
Replaced `-` with `_` in several OCSF entities, including `network_activity` in commit https://github.com/opencybersecurityalliance/kestrel-lang/pull/414/commits/15c9caec06163a26299565dfdf151fcf4509fcfa